account_invoice_ticket/README.rst

```diff
@@ -0,0 +1,28 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=========================
+Account Invoice Ticket
+=========================
```

Comment:
Theres an error
```md
======================
Account Invoice Ticket
======================
```

The markdown should match the line's length (22 characters).